lib/experimental/fonts-library/class-wp-font-family-utils.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Fonts Family Utils class.
+ *
+ * This file contains utils fot Font Family class.
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ * @since      X.X.X
+ */
+
+/**
+ * A class of utilities for working with the Font Library.
+ */
+class WP_Font_Family_Utils {
+
+	/**
+	 * Generates a filename for a font face asset.
+	 *
+	 * Creates a filename for a font face asset using font family, style, weight and extension information.
+	 *
+	 * @param array  $font_face The font face array containing 'fontFamily', 'fontStyle', and 'fontWeight' attributes.
+	 * @param string $url The URL of the font face asset, used to derive the file extension.
+	 * @param int    $i Optional counter for appending to the filename, default is 1.
+	 * @return string The generated filename for the font face asset.
+	 */
+	static public function get_filename_from_font_face( $font_face, $url, $i = 1 ) {
```

Comment:
The preferred approach (and for consistency within Core) is to start with the visibility keyword:

```suggestion
	public static function get_filename_from_font_face( $font_face, $url, $i = 1 ) {
```

This comment applies to each of the methods.

Reply:
Done! https://github.com/WordPress/gutenberg/pull/52704/commits/98faaff70aa36c275dc3d1403088097fede35b4b